css/styless.css

```diff
@@ -316,6 +320,7 @@ li {
   color: #fff;
   transition: all 0.5s ease-in;
   margin: auto;
+  border: 1px;
 }
```

Comment:
- [OPTIONAL] You can center your button text on the multi-post stories by using the "Justify-content: center;" property on the ".rbb-btn" class.

before the edit:
![image](https://user-images.githubusercontent.com/99830261/193172601-c1e617bc-6414-4430-9cb3-e993b92dabb3.png)

after the edit:
![image](https://user-images.githubusercontent.com/99830261/193171291-6275015c-9274-4301-a7e2-86dede47340c.png)